### DISCUSSION TITLE
OIDC Implementation - Public or Private Downloads?

### DISCUSSION THREAD

**Discussion by mrnoisytiger** [Forceu/Gokapi#186] [Q&A] [answered]
Right now, I'm doing some pretty unusual `auth` with CF tunnels that allows me "split-auth" admin portions of the app (for uploads) and public portions of the app (for downloads). It would be interesting if I could integrate my Azure AD // EntraID credentials instead since that's what I use for SSO across my environment. 

However, as I use Gokapi like a self-hosted file share, when people request that I send large files that can't fit over email, it's important that they would still be able to access public downloads without authentication. 

Does the current OIDC implementation create this "split auth" scenario that allows for public downloads, or is the entire app behind authentication? 

Thanks!

**Comment by Forceu** [comment, marked as answer]
Hi, sorry for the late reply. No, the download portion is public and the admin section requires authentication.

**Reply by mrnoisytiger** [reply]
Thank you!
